services/cii-best-practices/cii-best-practices.tester.js

```diff
@@ -0,0 +1,252 @@
+'use strict'
+
+const Joi = require('joi')
+const t = require('../create-service-tester')()
+module.exports = t
+
+t.create('live: level known project')
+  .get(`/level/1.json`)
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: 'cii',
+      value: Joi.string(),
```

Comment:
The three first live tests would benefit from stronger validation on the `value` parameter. Currently, if the tested projects were deleted or if the API changed and we were no longer able to parse the response, the tests would still pass as the value would still be a string. You can have a look at [test-validators.js](https://github.com/badges/shields/blob/master/services/test-validators.js) for some inspiration. 😉

Reply:
🤦‍♂️ - good catch!